app/views/diaper_drives/index.html.erb

```diff
@@ -79,6 +79,7 @@
                 <th>Diaper Drive Name</th>
                 <th>Start Date</th>
                 <th>End Date</th>
+                <th>Virtual?<th>
```

Comment:
```suggestion
                <th>Held Virtually?<th>
```
I think this might a little more clear in the long run. Outside of the context of this issue, it might not be obvious what 'Virtual?' means in this case. I suspect that not all of our users know what virtual means, this is a good opportunity to educate users that "Yes you can input diaper drives that are held virtually?"